docs/user/feature-flags.rst

```diff
@@ -18,6 +18,15 @@ This helps users to pin dependencies on conda and to improve build time.
 
 ``DONT_OVERWRITE_SPHINX_CONTEXT``: :featureflags:`DONT_OVERWRITE_SPHINX_CONTEXT`
 
+``SKIP_SPHINX_HTML_THEME_PATH``: :featureflags:`SKIP_SPHINX_HTML_THEME_PATH`
```

Comment:
We stopped advertising feature flags like this. If a feature flag is in this list, it should be a configuration option instead, something the user can opt into without requiring support time.

Reply:
That sounds good. By "configuration option", do you mean a variable that's defined in `conf.py` and then we check `globals()` ?

Reply:
Not `conf.py`, but in our own configuration file, `.readthedocs.yaml`. We have a number of issues open about deprecation of feature flags, but this particular doc was a focus of most of our removals.

I'm not suggesting this fix is implemented as a configuration option though. Just that feature flags should be reserved for internal use only.

Reply:
Right, got it, yeah `.readthedocs.yaml` is overkill, I don't expect anyone to really need the feature flag, it's more like a safety net so behavior remains fully configurable. Hopefully this feature flag will be unnecessary soon, as the number of builds for _very old_ is in decline.